cmd/ndm_daemonset/probe/mountprobe.go

```diff
@@ -106,3 +128,36 @@ func (mp *mountProbe) FillBlockDeviceDetails(blockDevice *blockdevice.BlockDevic
 		blockDevice.FSInfo.FileSystem = basicMountInfo.FileSystem
 	}
 }
+
+func (mp *mountProbe) setupEpoll(events ...epoll.EventType) error {
```

Comment:
Why are these args required. We will be always checking for EPOLLERR and EPOLLPRI right?

Reply:
yes. had put these for testing purposes but they aren't required. will be removed in the next revision.